ecms_base/modules/custom/ecms_languages/src/Form/EcmsLanguageSettings.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\ecms_languages\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Language\LanguageManager;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a configuration form to exclude languages from the switcher.
+ */
+class EcmsLanguageSettings extends ConfigFormBase {
+
+  /**
+   * Language manager service.
+   *
+   * @var \Drupal\Core\Language\LanguageManager
+   */
+  protected $languageManager;
+
+  /**
+   * Class constructor.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The plugin factory to create the condition plugin.
```

Comment:
Nit: The comment doesn't match the variable. Maybe `the config.factory service.`. I personally always comment these with the actual service name, mostly so I remember the service name.